django_app/settings.py

```diff
@@ -150,7 +150,7 @@
     if os.getenv("DATABASE_URL", None) is None:
         raise Exception("DATABASE_URL environment variable not defined")
     DATABASES = {
-        "default": dj_database_url.parse(os.environ.get("DATABASE_URL")),
+        "default": dj_database_url.parse(os.environ.get("DATABASE_URL"), conn_max_age=6000),
```

Comment:
According to my understanding connection max age should be 60, it might be show too many connection errors.

Reply:
@robel-milashu thoughts?

Reply:
It depends on the network traffic, lets do it 60 and we can raise if the number of users raise or if we got network traffic issue

Reply:
@robel-milashu As you see in the README.md, this app will never have more than 10 simultaneous users. This is an application used to collect scientific data for audio AI research